domains/bennu.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "bennu99",
+        "email": "bennu9909@gmail.com"
+    },
+    "record": {
+        "URL": "www.bennu.is-a.dev"
```

Comment:
- missing protocol
- you can't redirect the website to itself

Reply:
com"
    },
    "record": {
        "URL": "https://visual-arts.ai" 
    }
}